### DISCUSSION TITLE
Using Proxy with new API Token Format

### DISCUSSION THREAD

**Discussion by shawncarr** [Unleash/unleash#1863] [Q&A] [answered]
I am interested in using the unleash proxy but due to the format of API Tokens with 4.3, it seems as if you would need a proxy per environment as the API Token provided to the proxy on start-up would only be able to read features for the provided Token Environment. 

Is that correct?

**Comment by chriswk** [comment, marked as answer]
Yes, with these new tokens you will need a proxy per environment, since you can only use one key per proxy for authenticating against the Unleash server.
